File: Gosei.EmployeeQualificationManagement/src/Gosei.EmployeeQualificationManagement.Application.Contracts/Dtos/Employees/EmployeeQualification/EmployeeQualificationResponse.cs
using System;
using Volo.Abp.Application.Dtos;

namespace Gosei.EmployeeQualificationManagement.Dtos.Employees.EmployeeQualification
{
    public class EmployeeQualificationResponse : EntityDto<Guid>
    {
        public string Institution { get; set; }
        public string City { get; set; }
        public DateTime? ValidFrom { get; set; }
        public DateTime? ValidTo { get; set; }

        // qualification
        public string QualificationName { get; set; }
    }
}

File: frontend/src/types/response/employeeQualification.response.ts
import { PaginatedResponse } from "../common"

export type EmployeeQualificationPaginatedResponse = PaginatedResponse<EmployeeQualificationResponse>;

export type EmployeeQualificationResponse = {
    id?: string,
    institution: string,
    city: string | null,
    validFrom: Date | null,
    validTo: Date | null,
    qualificationName: string
}